lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -137,6 +142,15 @@ negotiate_auto_portchannel:
       config_set_append: "%s negotiate auto"
       default_value: true
 
+port_channel:
+  kind: int
+  config_get: "show running interface all"
+  config_get_token: ['/^interface %s$/i', '/^channel-group (\d+)$/']
+  config_set: ["interface %s", "%s channel-group %s %s"]
+
+port_channel_destroy:
```

Comment:
You don't need this. The destroy entry already does that:

https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/interface.yaml#L34